lib/hermann/provider/java_simple_consumer.rb

```diff
@@ -38,16 +40,16 @@ class JavaSimpleConsumer
       # @option opts [Boolean] :do_retry Retry consume attempts if exceptions are thrown, defaults to true
       # @option opts [Fixnum]  :max_retries Number of max_retries to retry #consume when it throws an exception
       # @option opts [Logger]  :logger Pass in a Logger
+      # @option opts [Other]   other opts from kafka
       def initialize(zookeepers, groupId, topic, opts={})
-        config            = create_config(zookeepers, groupId)
-        @consumer         = ConsumerUtil::Consumer.createJavaConsumerConnector(config)
         @topic            = topic
-
         options           = DEFAULT_CONSUMER_OPTIONS.merge(opts)
-        @backoff_time_sec = options[:backoff_time_sec]
-        @do_retry         = options[:do_retry]
-        @max_retries      = options[:max_retries]
-        @logger           = options[:logger]
+        @backoff_time_sec = options.delete(:backoff_time_sec)
```

Comment:
Deleting because we do not want to pass these options into the create_config.  So they are stripped out and the remaining options should be intended for the create_config method.